core/src/main/java/org/bitcoinj/store/PostgresFullPrunedBlockStore.java

```diff
@@ -81,7 +81,7 @@
             ")\n";
```

Comment:
There is one more occurence in the line above. Does that need to be escaped, too?

Reply:
yes, thanks